packages/openactive-integration-tests/test/helpers/flow-stages/b.js

```diff
@@ -35,7 +35,7 @@ async function runB({ templateRef, uuid, sellerId, orderItems, totalPaymentDue,
     totalPaymentDue,
     orderProposalVersion,
   };
-  const response = await requestHelper.putOrder(uuid, params, templateRef);
+  const response = await requestHelper.putOrder(uuid, params, null, templateRef);
```

Comment:
There's a new `brokerRole` arg in these 3 requestHelper functions, so templates weren't working 😧

Reply:
Good catch!